src/main/java/appeng/me/GridConnection.java

```diff
@@ -62,7 +62,7 @@ public GridConnection( final IGridNode aNode, final IGridNode bNode, final AEPar
 
 		if( Platform.securityCheck( a, b ) )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Nullcheck of aNode at line 77 of value previously dereferenced in new appeng.me.GridConnection(IGridNode, IGridNode, AEPartLocation) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=findbugs%3ARCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Nullcheck of bNode at line 77 of value previously dereferenced in new appeng.me.GridConnection(IGridNode, IGridNode, AEPartLocation) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=findbugs%3ARCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE)